includes/view/User_view.php

```diff
@@ -348,21 +348,21 @@ function User_view_myshift(Shift $shift, $user_source, $its_me)
         'comment'    => '',
         'start'      => $shift->start,
         'end'        => $shift->end,
-        'freeloaded' => $shift->freeloaded,
+        'freeloaded' => $shift->freeloaded_by,
     ];
 
     if ($its_me) {
         $myshift['comment'] = htmlspecialchars($shift->user_comment);
     }
 
-    if ($shift->freeloaded) {
+    if ($shift->freeloaded_by) {
         $myshift['duration'] = '<p class="text-danger"><s>'
             . sprintf('%.2f', ($shift->end->timestamp - $shift->start->timestamp) / 3600) . '&nbsp;h'
             . '</s></p>';
         if (auth()->can('user_shifts_admin') || $supporter) {
             $myshift['comment'] .= '<br />'
                 . '<p class="text-danger">'
-                . __('Freeloaded') . ': ' . htmlspecialchars($shift->freeloaded_comment)
+                . __('Freeloaded by %s', [User_Nick_render($shift->freeloadedBy)]) . ' : ' . htmlspecialchars($shift->freeloaded_comment)
```

Comment:
```suggestion
                . __('Freeloaded by %s', [User_Nick_render(User::find($shift->freeloaded_by))]) . ' : ' . htmlspecialchars($shift->freeloaded_comment)
```

freeloaded by can't be used here directly as the shift model is a "legacy" meshup of the actual Shift model with some ShiftEntry values added on top. `freeloaded_by` exists for that reason but is a plain integer id